### ISSUE TITLE
Does the training set contain clean validation set for noisy label experiments?

### ISSUE THREAD

**Issue by lanzhuzhu** [uber-research/learning-to-reweight-examples#1]
Hi, thank you for open-sourcing this nice library for learning-to-reweight-examples, interesting work!
I have a question about how the clean validation set was used.
According to the last sentence of page2 in your paper, "the training set contains the validation set", so does this mean that you add the clean validation set to the training set for your model? Since other base models are fine-tuned with the clean validation set, if it means that their training set does not contain clean validation set?

**Comment by mengye-ren** [comment]
Hi, the clean validation set is split from the training set.

**Comment by lanzhuzhu** [comment]
> Hi, the clean validation set is split from the training set.

Thanks a lot for your reply!
